SQL/paradise_schema.sql

```diff
@@ -249,6 +249,9 @@ CREATE TABLE `player` (
   `nanoui_fancy` smallint(4) DEFAULT '1',
   `show_ghostitem_attack` smallint(4) DEFAULT '1',
   `lastchangelog` varchar(32) NOT NULL,
+  `space_parallax` smallint(4) DEFAULT '1',
+  `space_dust` smallint(4) DEFAULT '1',
+  `parallax_speed` smallint(4) DEFAULT '2',
```

Comment:
Make this a float, there's currently an issue on /vg/ due to this being an integer (decimals don't save)